### PR TITLE
10_4_3 Anagram 2

### PR DESCRIPTION
**Pull request by AleksandraYanum**

### CODE REVIEW

File: 10_dictionary/10_4_dictionary_tasks/3_anagram_2.py [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+        result = word_letter_amount == {}
+
+    return result
+
+
+def main():
+    string_1, string_2 = ''.join(c.lower() for c in input() if c.isalpha()), \
+                         ''.join(c.lower() for c in input() if c.isalpha())
+
+    output = POSITIVE_OUTPUT if is_anagram(string_1, string_2) else NEGATIVE_OUTPUT
```
Comment (YanumVlad) [inline 10_dictionary/10_4_dictionary_tasks/3_anagram_2.py:28]: Extract business logic

File: 10_dictionary/10_4_dictionary_tasks/3_anagram_2.py [hunk 2]
```diff
@@ -0,0 +1,38 @@
+POSITIVE_OUTPUT = 'YES'
+NEGATIVE_OUTPUT = 'NO'
+
+
+def is_anagram(string_1, string_2):
+    result = False
+    word_letter_amount = {}
```
Comment (YanumVlad) [inline 10_dictionary/10_4_dictionary_tasks/3_anagram_2.py:7]: Inconsistency: word / string, letter / char

**Review by YanumVlad — DISMISSED** [review]

File: 10_dictionary/10_4_dictionary_tasks/3_anagram_2.py (hunk 2 above)
Comment (AleksandraYanum) [reply 10_dictionary/10_4_dictionary_tasks/3_anagram_2.py]: Done

File: 10_dictionary/10_4_dictionary_tasks/3_anagram_2.py (hunk 1 above)
Comment (AleksandraYanum) [reply 10_dictionary/10_4_dictionary_tasks/3_anagram_2.py]: Done

**Review by YanumVlad — APPROVED** [review]

**Pull request merged**